File: Using_Java/Simple-E-Commerce/src/ShippingService.java
import java.util.List;

public class ShippingService {
    public void shippingItems(List<IShippable> list) {
        System.out.println("\nShipping the following items:");
        for (IShippable item : list) {
            System.out.println("- " + item.getName() + ", Weight: " + item.getWeight() + " kg");
        }
    }
} 

File: Using_C#/Simple-E-Commerce/ShippingService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Simple_E_Commerce
{
    class ShippingService
    {
        public void ShippingItems(List<IShippable> list)
        {
            Console.WriteLine("\nShipping the following items:");
            foreach (var item in list)
                Console.WriteLine($"- {item.GetName()}, Weight: {item.GetWeight()} kg");
        }
    }
}
